### PR TITLE
fix placeholder

### PR DESCRIPTION
**Pull request by anjmao**
fixes https://github.com/ng-select/ng-select/issues/297

### CODE REVIEW

File: src/ng-select/ng-select.component.ts [hunk 1]
```diff
@@ -59,7 +59,8 @@ export type DropdownPosition = 'bottom' | 'top' | 'auto';
         '[class.top]': 'currentDropdownPosition === "top"',
         '[class.bottom]': 'currentDropdownPosition === "bottom"',
         '[class.ng-single]': '!multiple',
-        '[class.ng-selected]': 'hasValue'
```
Comment (varnastadeus) [inline src/ng-select/ng-select.component.ts]: have you tried with `@HostBinding` decorator?

Comment (anjmao) [reply src/ng-select/ng-select.component.ts]: yes, same "shit"

**Comment by anjmao** [comment]
I got answer from other developer
> As far i know that's by desing in Renderer2.
> 
> Calling detectChanges does not have any effect because host bindings are part of parent view.
> Hi view doesn't contain expression for updating class.has-value in your example
> Angular will update it during executing updateRenderer function for parent view.

File: src/ng-select/ng-select.component.ts (hunk 1 above)
Comment (anjmao) [reply src/ng-select/ng-select.component.ts]: Move this to ng-control instead of host class

**Comment by abindalSchuco** [comment]
How can I get this fix? we're using Angular version 7.2, I tried updating ng-select version to 2.20.0, But that doesn't fix this issue.

**Pull request merged**